app/helpers/application_helper.rb

```diff
@@ -1,2 +1,4 @@
 module ApplicationHelper
+	require "codeclimate-test-reporter"
+CodeClimate::TestReporter.start
```

Comment:
Indentation is a bit off here.

If you run `rubocop -a` in your terminal and then commit the changes. It will autocorrect style issues :)